astropy/coordinates/transformations.py

```diff
@@ -674,6 +674,100 @@ def deco(func):
             return func
         return deco
 
+    def _create_direct_transform(self, fromsys, viasys, tosys, priority=1):
```

Comment:
I agree that the name is not clear.  It really should be `_create_and_register_direct_transform()`, but I thought that would be too long...

Reply:
I'm not sure about the `direct` part either. I'd vote for the more explicit function name. Since this is a method on the frame transfer graph I think `_register...` works, but how about `_register_twostep_transform`?

Reply:
After mulling it over with fresh eyes, I decided to change the name to ``_add_merged_transform``.